src/actions/receive-profile.js

```diff
@@ -94,6 +95,16 @@ export function viewProfile(
       selectedThreadIndex,
       pathInZipFile,
     });
+
+    // Update ProfileSharingStatus from profile metadata.
+    // If the profiler is not published before, networkURLsRemoved
+    // will be undefined.
+    dispatch(
+      setProfileSharingStatus({
+        sharedWithUrls: profile.meta.networkURLsRemoved === false,
+        sharedWithoutUrls: profile.meta.networkURLsRemoved === true,
+      })
+    );
```

Comment:
I'm not sure if this dispatch is correct or not. I wrote this because we needed to update `profileSharingStatus` when we dispatch `VIEW_PROFILE`. But this change added a `{ phase: 'DATA_LOADED' },` line to tests in src/test/store/receive-profile.test.js below.